### ISSUE TITLE
Uncaught Syntax Error

### ISSUE THREAD

**Issue by skemuel007** [ashinzekene/angular4-paystack#25]
I get this message on chrome
> `Uncaught SyntaxError: Unexpected token '<'`
But works fine on Mozilla

Another issue - **Payment modal launches only once**

Please I need your help

Stanley-Kemuel Lloyd Salvation

**Comment by ashinzekene** [comment]
Can you explain how you get the syntax error?

**Comment by skemuel007** [comment]
This happens when I want to make payment, it occurs only in chrome, the spinner before modal launch keeps rolling without launch ing the payment modal, then the error above is displayed.

**Comment by skemuel007** [comment]
![15867347634233999572423664722900](https://user-images.githubusercontent.com/11774340/79082622-5b73df00-7d1f-11ea-80bf-5735d02da873.jpg)
The error screen shot

**Comment by skemuel007** [comment]
![15867354642838831360355204700768](https://user-images.githubusercontent.com/11774340/79082768-0507a000-7d21-11ea-8d3e-c83669b6278f.jpg)
Chrome not working

**Comment by ashinzekene** [comment]
This is most probably from your code, try debugging the source
